lib/api/controllers/serverController.js

```diff
@@ -110,6 +111,21 @@ class ServerController extends NativeController {
     return config;
   }
 
+  /**
+   * Returns the Kuzzle capabilities
+   * @returns {Promise<Object>}
+   */
+  async capabilities () {
+    const config = JSON.parse(JSON.stringify(global.kuzzle.config));
+    const publicApi = await this.publicApi();
+
+    return {
+      limits: config.limits,
+      routes: publicApi,
+      services: config.services,
```

Comment:
You shouldn't expose the entire services configuration since it can contains sensitive information such as ES credentials.  

Just exposing the services backend names should be enough info